From bcaab4099232c51bae0a8bec44dcb66166691ba8 Mon Sep 17 00:00:00 2001
From: Sahil Malhotra <sahil.malhotra@nxp.com>
Date: Sat, 3 Aug 2019 09:45:28 +0530
Subject: [PATCH 554/741] arm64: dts: fsl: add optee node for ls1028

commit e6437f50ec614cf4ef243ab2e418cea4e2c90102 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

For enabling OP-TEE on LS1028, need to add optee node
in DTS

Signed-off-by: Sahil Malhotra <sahil.malhotra@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index a99adb3eed4f..ba5a848d0474 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -766,6 +766,13 @@
                 };
 	};
 
+        firmware {
+                optee {
+                        compatible = "linaro,optee-tz";
+                        method = "smc";
+                };
+        };
+
         display0: malidp@f080000 {
                 compatible = "arm,mali-dp500";
                 reg = <0x0 0xf080000 0x0 0x10000>;
-- 
2.17.1

